_posts/2022-10-25-OpenSearch-Project-voice-and-tone.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: post
+title:  "From the editor's desk: OpenSearch Project voice and tone"
+authors: 
+  - nbower
+date: 
+categories: 
+  - community
+---
+
+In the [first installment]( https://opensearch.org/blog/community/2022/08/New-series-From-the-editors-desk/) of our *From the editor’s desk* series, we focused on the OpenSearch Project and inclusion, a subject that is foundational to who we are and how we talk to each other as a community. For the second installment, we’d like to focus on another subject related to community interaction: voice and tone.
+
+In response to feedback received during [OpenSearchCon 2022]( https://opensearch.org/blog/community/2022/05/opensearchcon/), we have updated our guidance on the voice and tone of the OpenSearch Project in our [OpenSearch Project style guidelines]( https://github.com/opensearch-project/documentation-website/blob/main/STYLE_GUIDE.md). We have also added new guidance on OpenSearch Project naming conventions and brand personality traits. Check out this new and updated guidance in the following sections, and feel free to let us know your thoughts.
+
+## Naming conventions
+
+The following naming conventions should be observed in OpenSearch Project content:
+
+* Capitalize both words when referring to the *OpenSearch Project*. The OpenSearch Project has three products:  OpenSearch, OpenSearch Dashboards, and Data Prepper.
```

Comment:
Casting my vote for "the OpenSearch Project," one, to be consistent with how the project name has been used up to this point in time, and secondly because I don't think I would consider it a product in the traditional sense. To me it's broader in scope.

Reply:
Agree with James.